modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryServiceTest.java

```diff
@@ -54,7 +54,7 @@ public class AccountEntryServiceTest {
 	@Before
 	public void setUp() throws Exception {
 		_accountEntry1 = AccountEntryTestUtil.addAccountEntry();
-		_company = CompanyTestUtil.addCompany();
+		_company = _companyLocalService.getCompany(TestPropsValues.getCompanyId());
```

Comment:
Leave as it is, in this case we need a new company because we are checking permissions